tools/autograd/templates/VariableType.h

```diff
@@ -49,7 +51,7 @@ struct VariableType final : public at::Type {
   virtual std::unique_ptr<at::Storage> unsafeStorageFromTH(void * th_pointer, bool retain) const override;
   virtual at::Tensor unsafeTensorFromTH(void * th_pointer, bool retain) const override;
 
-  static at::Type* getType(const at::Type& baseType);
+  TORCH_API static at::Type* getType(const at::Type& baseType);
```

Comment:
Maybe we should just `TORCH_API` the entire class?

Reply:
does that work? I can shoot off a test run and switch to that if it passes

Reply:
kicked off this test run to check https://github.com/pytorch/pytorch/pull/9429